lib/shared/src/guardrails/client.ts

```
@@ -2,11 +2,17 @@ import type { SourcegraphGraphQLAPIClient } from '../sourcegraph-api/graphql'
 import { isError } from '../utils'
 
 import type { Attribution, Guardrails } from '.'
+import { ConfigFeaturesSingleton } from '../sourcegraph-api/graphql/client'
 
 export class SourcegraphGuardrailsClient implements Guardrails {
     constructor(private client: SourcegraphGraphQLAPIClient) {}
 
     public async searchAttribution(snippet: string): Promise<Attribution | Error> {
```

unrelated to your pr but returning an error seems a bit odd. Should we throw the error instead?

---

I saw it used a couple times here and there. Happy to follow up to clean it up.

https://sourcegraph.com/search?q=repo:^github\.com/sourcegraph/cody%24++|+Error%3E&patternType=standard&sm=1

---

https://github.com/sourcegraph/sourcegraph/issues/59758